jenkins-build.cmd

```diff
@@ -107,11 +107,42 @@ echo Building the test tree using configuration: %BUILD_PROFILE%
 @if ERRORLEVEL 1 echo Error: VS integration unit tests build failed && goto :failure
 
 echo Running update scripts
+call src\update.cmd %BUILD_PROFILE% -ngen
+call vsintegration\update-vsintegration.cmd %BUILD_PROFILE%
 
+REM Build the tools that are used for testing
 @echo on
 call tests/BuildTestTools.cmd %BUILD_PROFILE% 
 @if ERRORLEVEL 1 echo Error: 'BuildTestTools.cmd %BUILD_PROFILE%' failed && goto :failure
 
+REM Actually run the tests
+
+cd tests
```

Comment:
Since Jenkins is much more powerful, we should aim towards running all tests, not just the smoke ones.